src/main/java/com/example/demo/OrderComponent/API/Endpoints/OrderController.java

```diff
@@ -0,0 +1,97 @@
+package com.example.demo.OrderComponent.API.Endpoints;
+
+import com.example.demo.OrderComponent.API.DTOs.ModifyOrderDTO;
+import com.example.demo.OrderComponent.API.DTOs.OrderDTO;
+import com.example.demo.OrderComponent.ApplicationServices.OrderApplicationService;
+import jakarta.persistence.EntityNotFoundException;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/v1/orders")
+public class OrderController {
+    private final OrderApplicationService orderApplicationService;
+
+    public OrderController(OrderApplicationService orderApplicationService) {
+        this.orderApplicationService = orderApplicationService;
+    }
+
+    @PostMapping
+    public ResponseEntity<?> createOrder(@RequestBody OrderDTO createOrderDTO) {
```

Comment:
@NotNull

Reply:
@NotNull is for the params, not the function